datalad/support/gitrepo.py

```diff
@@ -1390,14 +1390,13 @@ def count_objects(self):
         """return dictionary with count, size(in KiB) information of git objects
         """
         try:
-            count_str, err = Runner().run(
-                ['git', 'count-objects', '-v'],
-                log_stderr=True, log_stdout=True)
+            count_cmd = ['git', 'count-objects', '-v']
+            count_str, err = self._git_custom_command('', count_cmd)
             count_list = count_str.replace(': ', '\n').split('\n')
             count = {key: int(value) for (key, value) in zip(count_list[0::2], count_list[1::2]) if key and value}
             return count
         except CommandError:
-            raise(InvalidGitRepositoryError)
+            raise
```

Comment:
Well, transforming a general `CommandError` into something more meaningful isn't a bad idea from my point of view. I plan to do such transformations anyway. `GitRepo` should warp `git` and provide a python interface. To me this includes proper meaningful exceptions eventually.

But it would need a more specific evaluation of the `CommandError` here in order to be sure, that `InvalidGitRepositoryError` is the correct one to choose. (check error code and stderr)